godot-codegen/src/api_parser.rs

```diff
@@ -241,15 +241,17 @@ impl MethodReturn {
 // ----------------------------------------------------------------------------------------------------------------------------------------------
 // Implementation
 
-pub fn load_extension_api(watch: &mut godot_bindings::StopWatch) -> (ExtensionApi, &'static str) {
+pub fn load_extension_api(watch: &mut godot_bindings::StopWatch) -> (ExtensionApi, [&'static str; 2]) {
     // For float/double inference, see:
     // * https://github.com/godotengine/godot-proposals/issues/892
     // * https://github.com/godotengine/godot-cpp/pull/728
-    #[cfg(feature = "double-precision")]
-    let build_config = "double_64"; // TODO infer this
-    #[cfg(not(feature = "double-precision"))]
-    let build_config = "float_64"; // TODO infer this
-
+    let build_config: [&'static str; 2] = {
+        if cfg!(feature = "double-precision") {
+            ["double_32", "double_64"]
+        } else {
+            ["float_32", "float_64"]
+        }
+    };
```

Comment:
Thanks for explanation!

Could you maybe add that link with a short explanation above this snippet? That way, the design decision is documented and people don't wonder why 32/64 is not already determined at this point (like me) 🙂